src/views/AuthorityView/AuthorityView.js

```diff
@@ -129,38 +161,102 @@ const AuthorityView = ({
     return marcSourceClone;
   };
 
+  const onSubmit = () => {
+    deleteItem(authority.data.id);
+    setRequestDeletion(false);
+  };
+
   return (
-    <KeyShortCutsWrapper
-      onEdit={redirectToQuickMarcEditPage}
-      canEdit={hasEditPermission()}
-    >
-      <div data-testid="authority-marc-view">
-        <MarcView
-          paneTitle={authority.data.headingRef}
-          paneSub={intl.formatMessage({
-            id: 'ui-marc-authorities.authorityRecordSubtitle',
-          }, {
-            heading: authority.data.headingType,
-            lastUpdatedDate: intl.formatDate(marcSource.data.metadata.updatedDate),
-          })}
-          isPaneset={false}
-          marcTitle={intl.formatMessage({ id: 'ui-marc-authorities.marcHeading' })}
-          marc={markHighlightedFields().data}
-          onClose={onClose}
-          lastMenu={(
-            <IfPermission perm="ui-marc-authorities.authority-record.edit">
-              <Button
-                buttonStyle="primary"
-                marginBottom0
-                onClick={redirectToQuickMarcEditPage}
-              >
-                <FormattedMessage id="ui-marc-authorities.authority-record.edit" />
-              </Button>
-            </IfPermission>
-           )}
-        />
-      </div>
-    </KeyShortCutsWrapper>
+    <>
+      <ConfirmationModal
```

Comment:
Please put this component inside `<div data-testid="authority-marc-view">` at the end. This will minimize changes to markup and we usually put modals at the end of markup